.github/ISSUE_TEMPLATE/feature_enhancement.yaml

```
@@ -0,0 +1,38 @@
+# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Propose a feature enhancement
+description: Propose a new feature or an enhancement for Prefect
+labels: [requires triage]
+body:
+  - type: textarea
+    attributes:
+      label: Describe the proposed behavior
+      description: A clear and concise description of what new feature or behavior you would like to see
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Describe the current behavior
+      description: A clear and concise description of what Prefect currently does
+    validations:
+      required: true
```

```suggestion
  - type: textarea
    attributes:
      label: Describe the difference between the proposed behavior and current behavior
      description: A clear and concise description of what new feature or behavior you would like to see
    validations:
      required: true

  - type: textarea
    attributes:
      label: Describe the current behavior
      description: A clear and concise description of what Prefect currently does
    validations:
      required: true
```
Can these be combined

---

I don't think this makes as much sense @ahuang11. We're asking them to describe the difference between the proposed behavior, but they haven't given us the proposed behavior. IMO it makes the most sense in the original structure.

---

Okay what about combining "Describe the proposed behavior" and "Example Use"

"Describe the proposed behavior and provide an example"

---

Do we have examples of this introducing a barrier to contributing? Having one section with text and then another section with the code example doesn't seem that painful, but I also don't have nearly as much experience as a lot of people here with how our community contributions tend to be.

---

I guess the good news is that this is super easy to change, so getting it perfect on the first try doesn't feel like a super high-priority thing for me. I'll go ahead and combine them for now, and we can change it if it feels like the right thing to do.

---

I just made the code example optional, which seems like a meet-in-the-middle option.

---

Thanks!